README.md

```diff
@@ -30,13 +30,13 @@
   <strong>
     <a href="https://github.com/signalfx/tracing-examples/tree/main/opentelemetry-tracing/opentelemetry-java-tracing">Examples</a>
     &nbsp;&nbsp;&bull;&nbsp;&nbsp;
-    <a href="docs/faq.md">FAQ</a>
+    <a href="https://docs.splunk.com/Observability/gdi/get-data-in/application/java/splunk-java-otel-distribution.html">About the distribution</a>
     &nbsp;&nbsp;&bull;&nbsp;&nbsp;
     <a href="SECURITY.md">Security</a>
     &nbsp;&nbsp;&bull;&nbsp;&nbsp;
-    <a href="docs/supported-libraries.md">Supported Libraries</a>
+    <a href="https://docs.splunk.com/Observability/gdi/get-data-in/application/java/java-otel-requirements.html">Supported Libraries</a>
```

Comment:
How do we make sure that this list is updated when we add something to this repo?

Reply:
We update it here and I'm mentioned. As per CODEOWNERS, I should stay abreast of all changes in the .md files here.